src/Symfony/Component/Console/Application.php

```diff
@@ -1133,9 +1133,8 @@ private function splitStringByWidth($string, $width)
             $lines[] = str_pad($line, $width);
             $line = $char;
         }
-        if ('' !== $line) {
```

Comment:
@yceruto i think this is trimming related in fact :)

```
throw new \Exception("\n\nline 1 with extra spaces        \nline 2\n\nline 4\n");
```

Before/After
![image](https://user-images.githubusercontent.com/1047696/30503373-34dd143c-9a6a-11e7-87e1-bb695ccec72a.png)

![image](https://user-images.githubusercontent.com/1047696/30503384-3c972e1a-9a6a-11e7-8a3c-23b30b3a33cd.png)

So here's a better test ;-)

Reply:
Done.

Reply:
I think the trimming should be preserved ;-)